docs/index.yml

```diff
@@ -23,34 +23,49 @@ highlightedContent:
       title: Office Add-ins platform overview
       url: overview/office-add-ins.md
     # Card 2
-    - itemType: quickstart
-      title: Set up an Excel add-in in 5 minutes
-      url: quickstarts/excel-quickstart-jquery.md
-    # Card 3
-    - itemType: quickstart
-      title: Set up an Outlook add-in in 5 minutes
-      url: quickstarts/outlook-quickstart.md
-    # Card 4
-    - itemType: quickstart
-      title: Set up a Word add-in in 5 minutes
-      url: quickstarts/word-quickstart.md
-    # Card 5
-    - itemType: sample
-      title: Samples for Office Add-ins
-      url: overview/office-add-in-code-samples.md
-    # Card 6
     - itemType: how-to-guide
       title: Office Add-ins Beginner's guide
       url: overview/learning-path-beginner.md
-    # Card 7
+    # Card 3
     - itemType: sample
-      title: Test and experiment with APIs using Script Lab
-      url: overview/explore-with-script-lab.md
-    # Card 8
-    - itemType: reference
-      title: Office JavaScript API reference documentation
-      url: /javascript/api/overview
+      title: Samples for Office Add-ins
+      url: overview/office-add-in-code-samples.md
 
+productDirectory:
```

Comment:
I understand that the `title` attribute is [optional](https://review.learn.microsoft.com/help/platform/navigation-how-create-hub-page?branch=main#product-directory-section-multi-link), but should we consider adding one to the `productDirectory` section to match the `conceptualContent` section? Apart from the addition of the icon to the `productDirectory` cards, these sections share the same format (versus that of the top-level section). Not a deal-breaker, but thought it was worth asking.

Reply:
Good point. I agree that a title there would help draw attention. I'll try to think of something fitting, but I'm open to suggestions.